### PR TITLE
Upgrade Cosmos SDK

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: x/clp/client/rest/query.go [hunk 1]
```diff
@@ -98,19 +101,22 @@ func getCLPAccountOutput(clp *clpTypes.CLP, ctx context.CoreContext, decoder aut
...
-	jsonOutput := fmt.Sprintf("{\"%v\":%v,\"%v\":%v,\"price\":%v}", baseCoinTicker, baseCoinAmount, clp.Ticker, tickerCoinAmount, price)
+	jsonOutput := fmt.Sprintf("{\"%v\":%v,\"%v\":%v,\"price\":%v}", baseCoinTicker, baseCoinAmount, clp.Ticker,
+		tickerCoinAmount, price)
 
 	return []byte(jsonOutput), nil
 }
 
 // query all Handler
-func QueryAllRequestHandlerFn(cdc *wire.Codec, decoder auth.AccountDecoder, ctx context.CoreContext, baseCoinTicker string) http.HandlerFunc {
+func queryClpsRequestHandlerFn(
```
[the remark this answers is not in the document]
Comment (musnit) [reply x/clp/client/rest/query.go]: yeh that cognitive complexity is way too high we really need to do something about it

**Comment by Jessicawatsonmiller** [comment]
@philipstanislaus it appears CircleCI has failed because it can’t find gometalinter; can you take a look at this again so all the CircleCI tests pass?

**Comment by Jessicawatsonmiller** [comment]
@escobyte can you review this PR?

File: networks/remote/ansible/inventory/digital_ocean.py [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+        parser.add_argument('--env', '-e', action='store_true', help='Display DO_API_TOKEN')
+        parser.add_argument('--api-token', '-a', action='store', help='DigitalOcean API Token')
+
+        self.args = parser.parse_args()
+
+        if self.args.api_token:
+            self.api_token = self.args.api_token
+
+        # Make --list default if none of the other commands are specified
+        if (not self.args.droplets and not self.args.regions and
```
[the remark this answers is not in the document]
Comment (philipstanislaus) [reply networks/remote/ansible/inventory/digital_ocean.py]: Code copied from upstream, might be adjusted later

**Comment by philipstanislaus** [comment]
Exactly @musnit :) Thanks for the review!

**Pull request merged**